services/supervisor/src/transactions.rs

```diff
@@ -185,10 +185,11 @@ impl SupervisorInterface for Supervisor {
         config_confirms.confirm(&propose_hash, author);
 
         let config_entry = ConfigProposalWithHash {
-            config_propose: propose,
+            config_propose: propose.clone(),
             propose_hash,
         };
-        schema.pending_proposal().set(config_entry);
+
+        schema.pending_propose_hashes().put(&propose.actual_from.0, config_entry);
```

Comment:
If you will create a variable for `actual_from` height, you can avoid `clone` above.